libs/knowledge-store/ragstack_knowledge_store/math.py

```diff
@@ -0,0 +1,42 @@
+"""Copied from langchain_community.utils.math"""
```

Comment:
maybe worth mentioning the commit and/or adding a link (thinking about hte future where we may want to see if there are bug fixes, etc.)

Reply:
Done. Even better would be to write our own code. Even if this is MIT licensed.